qemu/tests/netperf_stress.py

```diff
@@ -67,6 +68,11 @@ def netperf_test(duration):
     for server in netperf_server:
         s_info = {}
         if server in vms:
+            if params.get("os_type") == "windows":
+                if params.get_numeric("smp") > 32 or params.get_numeric("vcpu_maxcpus") > 32:
+                    params["smp"] = params["vcpu_maxcpus"] = 32
+                params["start_vm"] = "yes"
```

Comment:
Hi @heywji 

Windows guest will run as netperf client or server, seems you add start_vm=no for all windows guest, but only modify start_vm=yes when server's os_type is windows.
So I think you also need to update line130
```
for client in netperf_client:
        c_info = {}
        if client in vms:
            client_vm = env.get_vm(client)
            client_vm.verify_alive()
```